decisions/websre/0002-web-sre-service-documentation.md

```diff
@@ -0,0 +1,48 @@
+# 2. Web SRE Service Documentation
+
+Date: 2021-06-01
+Scope: Web SRE Team
+
+## Status
+
+Proposed
+
+## Context
+
+The Web SRE Team manages a number of services that vary in users, technologies used, setup, our commitment, infrastructure, process, etc. 
+
+Pre-existing documentation for these services, if it exists, also varies - in location, level of detail, scope, structure, presumed audience, etc.
+
+To help share knowledge between the Web SRE Team about these services' unique situations, a consistent baseline for our services documentation is a required first step.
+
+## Decision
+
+A Web SRE service here is defined as any codebase where we have some ownership over an aspect of this codebase being functional - e.g. the deployment process, automation, infrastructure running the service, observability of some aspect(s), or authoring the code ourselves. Web SRE services could be externally or internally-facing, including infrastructure services only Web SRE are aware of and leverage.
```

Comment:
I'm not sure I fully understand what you're proposing, sorry. You mean something like, "externally facing or internally facing (whether due to being a development environment or some other reason for being internal)"?

Reply:
(let me try and write out an example of something to add?) "Web SRE are aware of and leverage.  This may include non-prod environments for testing infrastructure code (staging k8s clusters to test building new clusters) and supporting services (log aggregation, alerting, etc) as well as non-prod application code.  All should be documented."

Reply:
Okay, will add.

Reply:
Added a form of what you put above to the doc. Removed the all must be documented part, because the next line in the ADR was already every Web SRE service has a service documentation page. Let me know if this works @duallain and thanks!

Reply:
LGTM, ty.  Will Resolve this conversation.